AnnoyancesFilter/sections/popups.txt

```diff
@@ -131,6 +131,8 @@ kanaloco.jp###rt_popup
 cyberlink.com#$##macDetectPopupModal { display: none !important; }
 cyberlink.com#$#.modal-backdrop { display: none !important; }
 cyberlink.com#$#.modal-open { overflow: auto !important; }
+hollywood.com#$#.laurelrun-campaign { display: none!important; }
+hollywood.com#$#html.om-position-popup body { overflow: auto!important; }
```

Comment:
Cannot reproduce.
If the rules still necessary, add space before `!important;`.
Is `html.om-position-popup` part required?